src/components/Interpretations/common/Message/MessageIconButton.js

```diff
@@ -51,6 +51,20 @@ const MessageIconButton = ({
                         color: ${colors.teal600};
                         font-weight: 500;
                     }
+                    .button:hover {
+                        color: ${colors.grey900};
+                    }
+                    .icon {
```

Comment:
Hmm, I can't see that any html elements have the icon property.

Reply:
🤔  good observation, I don't understand how come it works then...